colorizer_data/converter.py

```diff
@@ -81,15 +87,8 @@ def _make_frame(
     seg2d = scale_image(seg2d, scale)
     seg2d = seg2d.astype(np.uint32)
 
-    # Remap the frame image so the IDs are unique across the whole dataset.
-    seg_remapped, lut = remap_segmented_image(
-        seg2d,
-        frame,
-        config.object_id_column,
-    )
-
-    writer.write_image(seg_remapped, frame_number)
-    update_bounding_box_data(bounds_arr, seg_remapped)
```

Comment:
Note: Bounding box data is not being calculated automatically and written anymore. In the future, it will ideally be passed in as columns, as described in #79